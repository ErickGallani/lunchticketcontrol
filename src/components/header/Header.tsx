import * as React from 'react';
import * as PropTypes from 'prop-types';
import FlatButton from 'material-ui/FlatButton';
import './header.css';

export default class Header extends React.Component {
    constructor(props: any){
        super(props);
    }
    
    render() {
        return (
            <div className="header">
                <p>Header</p>
            </div>
        );
    }
}