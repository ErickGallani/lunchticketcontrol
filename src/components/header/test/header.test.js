import React, { Component } from 'react';
import Header from "../Header";
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

test('renders correctly headr', () => {
  const elem = shallow(<Header />);
  expect(shallowToJson(elem)).toMatchSnapshot();
});