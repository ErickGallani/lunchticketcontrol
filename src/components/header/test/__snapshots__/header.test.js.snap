// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly headr 1`] = `
<div
  className="header"
>
  <p>
    Header
  </p>
</div>
`;
